kubernetes/config-maps/concent-api-worker/local_settings.py.j2

```diff
@@ -18,7 +18,7 @@ DEBUG_INFO_IN_ERROR_RESPONSES = {{ allow_debug_info_in_error_responses }}
 
 LOGGING['handlers']['console']['level'] = DEBUG
 
-{% if signing_service_on_the_cluster %}
+{% if allow_signing_service_authentication %}
 USE_SIGNING_SERVICE = True
 {% endif %}
```

Comment:
You should set this to `False` explicitly instead of relying on a default.

Something like:
``` python
USE_SIGNING_SERVICE = {{ True if allow_signing_service_authentication else False }}
```
(not tested; you may have to use `"True"` instead of `True`).

Reply:
Done.